index.html

```diff
@@ -153,6 +119,84 @@ <h4 class="text-uppercase m-0">y Seas</h4>
             </div>
           </div>
         </div>
+        <div class="signup-section d-none" id="add-books">
+          <div class="row gx-4 gx-lg-5">
+            <div class="col-md-10 col-lg-8 mx-auto text-center">
+              <i class="far fa-paper-plane fa-2x mb-2 text-white"></i>
+              <h2 class="text-white mb-5">Add your books here!</h2>
+              <form
+                class="form-signup add-book"
+                id="contactForm"
+              >
+                <!-- book title input-->
+                <div class="row input-group-newsletter">
+                  <div class="col">
+                    <input
+                      class="form-control title"
+                      id="emailAddress"
+                      name="title"
+                      type="text"
+                      placeholder="Enter book title ..."
+                      aria-label="Enter book title ......"
+                    />
+                  </div>
+                </div>
+                <br />
+                <div class="row input-group-newsletter">
+                  <div class="col">
+                    <input
+                      class="form-control author"
+                      type="text"
+                      name="author"
+                      placeholder="Enter Author..."
+                      aria-label="Enter Author.."
+                    />
+                  </div>
```

Comment:
- [OPTIONAL] In your js function, it would be better to add a `resetForm` function to clear inputs after successfully submitting form data to improve the user experience.